src/views/EventsAttended/index.jsx

```diff
@@ -8,13 +8,6 @@ import { Link } from 'react-router-dom';
 import event from 'services/event';
 import { gordonColors } from 'theme';
 
-const style = {
```

Comment:
You removed this style, but on lines 33 and 55 you still have buttons that use this.  You should be able to just add the color='secondary' prop to the buttons to get the same effect